CI/unit_tests/agents/test_lymburn_model.py

```diff
@@ -0,0 +1,209 @@
+
+    def test_homing_force(self):
+        self.force_model.update_force_params(K_a=0, K_r=0, K_h=0, K_f=1, K_p=0)
+        self.force_model.update_force_params(K_h=1)
+
+        home_coll = Colloid(
+            pos=np.array([500.0, 500.0, 0]),
+            director=np.array([1.0, 0.0, 0.0]),
+            id=1,
+            velocity=np.array([10.0, 0.0, 0.0]),
+            type=0,
+        )
+
+        other_coll = Colloid(
+            pos=np.array([510.0, 500.0, 0.0]),
+            director=np.array([1.0, 0.0, 0.0]),
+            id=2,
+            velocity=np.array([0.0, 10.0, 0.0]),
+            type=0,
+        )
+        action = self.force_model.calc_action([home_coll, other_coll])
+        force_home_coll = action[0].force
+        force_other_coll = action[1].force
+        new_dir_other_coll = action[1].new_direction
+        vec_to_other_coll = self.force_model.home_pos - other_coll.pos
+
+        new_dir_other_coll /= np.linalg.norm(new_dir_other_coll)
+        vec_to_other_coll /= np.linalg.norm(vec_to_other_coll)
+        dot_prod = np.dot(new_dir_other_coll, vec_to_other_coll)
+
+        self.assertEqual(force_home_coll, 0)
+        self.assertGreater(force_other_coll, 0)
+        self.assertAlmostEqual(dot_prod, 1)
+
+    def test_friction_force(self):
+        self.force_model.update_force_params(K_a=0, K_r=0, K_h=0, K_f=1, K_p=0)
+
+        coll = Colloid(
+            pos=np.array([500.0, 500.0, 0]),
+            director=np.array([1.0, 0.0, 0.0]),
+            id=1,
+            velocity=np.array([30.0, 0.0, 0.0]),
+            type=0,
+        )
+        action = self.force_model.calc_action([coll])
+        force_coll = action[0].force
+        self.assertGreater(force_coll, 0)
+        self.assertEqual(
+            np.dot(action[0].new_direction, np.array([1.0, 0.0, 0.0])), -1.0
+        )
+
+    def test_predator_to_agent_force(self):
+        self.force_model.update_force_params(K_a=0, K_r=0, K_h=0, K_f=0, K_p=1)
+        self.force_model.update_pred_movement(swarmrl.agents.lymburn_model.no_force)
+        coll = Colloid(
+            pos=np.array([500.0, 500.0, 0]),
+            director=np.array([1.0, 0.0, 0.0]),
+            id=1,
+            velocity=np.array([10.0, 0.0, 0.0]),
+            type=0,
+        )
+        coll2 = Colloid(
+            pos=np.array([510.0, 500.0, 0]),
+            director=np.array([-1.0, 0.0, 0.0]),
+            id=2,
+            velocity=np.array([-10.0, 0.0, 0.0]),
+            type=0,
+        )
+        pred = Colloid(
+            pos=np.array([511.0, 500.0, 0]),
+            director=np.array([1.0, 0.0, 0.0]),
+            id=2,
+            velocity=np.array([-10.0, 0.0, 0.0]),
+            type=1,
+        )
+
+        action = self.force_model.calc_action([coll, coll2, pred])
+        force_coll = action[0].force
+        force_pred = action[-1].force
+        self.assertGreater(force_coll, 0)
+        self.assertEqual(force_pred, 0)
+
+
+class TestUtils(ut.TestCase):
```

Comment:
What is the vision cone test here for? I will add more comments to the main file, But I am confused as to why you are using vision cones at the moment.